test/components/submission/list.spec.js

```diff
@@ -374,6 +374,29 @@ describe('SubmissionList', () => {
         });
       });
 
+      describe('refreshing keys', () => {
+        it('sends request for encryption keys on submission refresh', () => {
+          testData.extendedProjects.createPast(1, {
+            key: testData.standardKeys.createPast(1).last(),
+            forms: 1
+          });
+          testData.extendedForms.createPast(1, {
+            submissions: 1
+          });
+          return load('/projects/1/forms/f/submissions', { root: false }, {
+            keys: () => [] // if there are 0 submissions, backend returns empty key array
+          })
+            .complete()
+            .request(component =>
+              component.get('#submission-list-refresh-button').trigger('click'))
+            .beforeAnyResponse(() => {
+              testData.extendedSubmissions.createPast(1, { status: 'notDecrypted' });
+            })
+            .respondWithData(() => testData.submissionOData(1, 0))
+            .respondWithData(() => testData.standardKeys.sorted());
```

Comment:
Ideas for additional assertions in this test
1. open the modal and check that the password input is shown
2. use `testRequests` to make sure the requests are to the right form/draft key endpoints (can specify null for first element if not necessary for the test)